common/styles/components/_buttons.scss

```diff
@@ -40,65 +40,16 @@
   }
 }
 
-.btn--primary {
-  border: 2px solid color('green');
-  background: color('green');
-  color: color('white');
-
-  &:not([disabled]):hover,
-  &:not([disabled]):focus {
-    border-color: color('black');
-    background: color('black');
-  }
-}
-
-.btn--secondary,
 .btn--tertiary {
   background: color('white');
   color: color('green');
-
-  &:not([disabled]):hover,
-  &:not([disabled]):focus {
-    background: color('green');
-    color: color('white');
-  }
-}
-
-.btn--secondary {
-  border: 2px solid color('green');
-}
-
-.btn--tertiary {
   padding: $spacing-unit+1 (2 * ($spacing-unit+1));
   border: 1px solid color('green');
-}
-
-.btn--primary-black {
-  border: none;
-  color: color('white');
-  background: transparent;
-  outline: none;
-  transition: all $transition-properties;
 
   &:not([disabled]):hover,
   &:not([disabled]):focus {
-    border-color: color('black');
-    background: color('yellow');
-    color: color('black');
+    background: color('green');
+    color: color('white');
   }
 }
 
-.btn--secondary-black {
-  border-color: color('charcoal');
-  color: color('smoke');
-  background: color('charcoal');
-  outline: none;
-  transition: all $transition-properties;
-
-  &:not([disabled]):hover,
-  &:not([disabled]):focus {
-    border-color: color('black');
-    background: color('yellow');
-    color: color('black');
-  }
-}
```

Comment:
How come we could get rid of all of this without it being an issue as we're using both buttons?

Reply:
There aren't any button-based references to `--secondary` in the codebase, and only one reference to `--tertiary` (in the `CopyUrl` component), so if we've made anything look like what was originally called a 'secondary' button, we must have done it in a cowboy fashion, apparently.